source/system-administrators/activities/setup-site-for-delivery.rst

```diff
@@ -75,8 +75,13 @@ Here is an example site's published repo git url when using a local repository:
       /opt/crafter/authoring/data/repos/sites/mysite/published
 
 .. note::
-  When using ``ssh``, you might see in the logs ``com.jcraft.jsch.JSchException: UnknownHostKey`` errors. These errors are common in Ubuntu, and are caused by known host
-  keys being stored in non-RSA format. Please follow the instructions in :ref:`crafter-studio-debugging-deployer-issues` under ``SSH Unknown Host`` to resolve them.
+  * When using ``ssh``, you might see in the logs ``com.jcraft.jsch.JSchException: UnknownHostKey`` errors. These errors are common in Ubuntu, and are caused by known host keys being stored in non-RSA format. Please follow the instructions in :ref:`crafter-studio-debugging-deployer-issues` under ``SSH Unknown Host`` to resolve them.
+
+  * ``git`` needs to be installed when using ssh to connect the authoring and delivery.
```

Comment:
Should we indicate that it's required on `authoring`?

Reply:
yes we need to indicate that it's required in authoring, not in delivery. I think the line should say `git needs to be installed in authoring when using SSH to connect the delivery to the authoring.